_config.yml

```diff
@@ -2,7 +2,7 @@ title: Oniro
 description: A starter template for a Jeykll site using the Just the Docs theme!
 theme: just-the-docs
 
-url: https://Yueqi77-amazing/eclipse-oniro4openharmony.github.io
+url: https://github.com/Yueqi777/eclipse-oniro4openharmony.github.io.git
```

Comment:
Both of these URLs need to be set correctly for the organization name instead of your user account at github.